test/spec_request.rb

```diff
@@ -114,32 +114,38 @@ class RackRequestTest < Minitest::Spec
     req = make_request \
       Rack::MockRequest.env_for("/", "HTTP_HOST" => "www2.example.org")
     req.host.must_equal "www2.example.org"
+    req.hostname.must_equal "www2.example.org"
 
     req = make_request \
       Rack::MockRequest.env_for("/", "SERVER_NAME" => "example.org", "SERVER_PORT" => "9292")
     req.host.must_equal "example.org"
+    req.hostname.must_equal "example.org"
 
     req = make_request \
       Rack::MockRequest.env_for("/", "HTTP_HOST" => "localhost:81", "HTTP_X_FORWARDED_HOST" => "example.org:9292")
     req.host.must_equal "example.org"
+    req.hostname.must_equal "example.org"
 
     req = make_request \
       Rack::MockRequest.env_for("/", "HTTP_HOST" => "localhost:81", "HTTP_X_FORWARDED_HOST" => "[2001:db8:cafe::17]:47011")
-    req.host.must_equal "2001:db8:cafe::17"
+    req.host.must_equal "[2001:db8:cafe::17]"
+    req.hostname.must_equal "2001:db8:cafe::17"
```

Comment:
This is likely to be a significant breaking change for IPv6 users.  The CHANGELOG should specifically call this out as a breaking change.

Reply:
Done.